test/builder/vcsim_test_context.go

```diff
@@ -648,18 +724,56 @@ func (c *TestContextForVCSim) setupK8sConfig(config VCSimTestConfig) {
 		data["StorageClassRequired"] = "true"
 
 		c.StorageClassName = "vcsim-default-storageclass"
-		// Use the hardcoded vcsim profile ID.
-		c.StorageProfileID = "aa6d5a82-1c88-45da-85d3-3d74b91a5bad"
+		c.StorageProfileID = "aa6d5a82-1c88-45da-85d3-3d74b91a5bad" // from vcsim
 
-		storageClass := &storagev1.StorageClass{
+		Expect(c.Client.Create(c, &storagev1.StorageClass{
 			ObjectMeta: metav1.ObjectMeta{
 				Name: c.StorageClassName,
 			},
 			Parameters: map[string]string{
 				"storagePolicyID": c.StorageProfileID,
 			},
-		}
-		Expect(c.Client.Create(c, storageClass)).To(Succeed())
+		})).To(Succeed())
+
+		c.EncryptedStorageClassName = "vm-encryption-policy"
+		c.EncryptedStorageProfileID = "4d5f673c-536f-11e6-beb8-9e71128cae77" // from vcsim
```

Comment:
Use the exported const `pbm/simulator.DefaultEncryptionProfileID` instead ?
We also could/should add one for ID used in `StorageProfileID` above.

Reply:
Yep. It did not exist when I wrote this. I'll switch to that :)